enginetest/enginetests.go

```diff
@@ -1786,6 +1786,170 @@ func TestCreateDatabase(t *testing.T, harness Harness) {
 	})
 }
 
+func TestPkOrdinals(t *testing.T, harness Harness) {
+	e := NewEngine(t, harness)
+	ctx := NewContext(harness)
+
+	TestQuery(t, harness, e, "CREATE DATABASE testdb", []sql.Row{{sql.OkResult{RowsAffected: 1}}}, nil, nil)
+
+	db, err := e.Analyzer.Catalog.Database("testdb")
+	require.NoError(t, err)
+
+	TestQuery(t, harness, e, "USE testdb", []sql.Row(nil), nil, nil)
+
+	require.Equal(t, ctx.GetCurrentDatabase(), "testdb")
+
+	t.Run("CREATE table out of order PKs", func(t *testing.T) {
+		TestQuery(t, harness, e, "CREATE TABLE a (x int, y int, primary key (y,x))", []sql.Row(nil), nil, nil)
```

Comment:
For queries like this where you aren't interested in asserting the result rows, use RunQuery instead

Reply:
👍